bundles/io/org.eclipse.smarthome.io.rest/src/main/java/org/eclipse/smarthome/io/rest/internal/filter/CorsFilter.java

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.io.rest.internal.filter;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.ext.Provider;
+
+import org.apache.commons.lang.StringUtils;
+
+import com.google.common.collect.Lists;
+
+/**
+ * A PostMatching filter used to add CORS HTTP headers on responses for requests with CORS
+ * headers.
+ *
+ * Based on http://www.w3.org/TR/cors
+ *
+ * This implementation does not allow specific request/response headers nor cookies (allowCredentials).
+ *
+ * @author Antoine Besnard
```

Comment:
Please change to something like "@author Antoine Besnard - Initial contribution"